scripts/phpize.m4

```diff
@@ -4,15 +4,20 @@ dnl Include external macro definitions before the AC_INIT to also remove
 dnl comments starting with # and empty newlines from the included files.
 m4_include([build/ax_check_compile_flag.m4])
 m4_include([build/ax_gcc_func_attribute.m4])
-m4_include([build/libtool.m4])
+m4_include([build/m4/libtool.m4])
+m4_include([build/m4/ltoptions.m4])
+m4_include([build/m4/ltsugar.m4])
+m4_include([build/m4/ltversion.m4])
+m4_include([build/m4/lt~obsolete.m4])
```

Comment:
This looks pretty fragile when it comes to supporting different libtool versions. Is it necessary to explicitly include all of these?